EntAppProject/src/main/java/com/jobook/EntAppProject/model/Address.java

```diff
@@ -1,54 +1,53 @@
-package com.jobook.EntAppProject.model;
-
-import org.springframework.stereotype.Component;
-
-public class Address {
-	private String city, state, zip, line1, line2;
-	private Integer id;
-
-	public Address(String city, String state, String zip, String line1, String line2, Integer id) {
-		super();

```

Comment:
As the pull request comment mentions, having `super()` in a class that only extends `Object` by default is unnecessary as that initialization/association is created automatically when the program is run. This change is present in many of the models.